api/tests/Unit/Entity/Repository/UserRepositoryTest.php

```diff
@@ -126,6 +126,180 @@ public function findActiveLaysInLastYear()
         self::assertNotContains($inactiveUserTwo->getId(), $resultsUserIds);
     }
 
+    /** @test */
+    public function getAllAdminAccounts()
```

Comment:
More a for your info, you can definitely use `@test ` annotation, owever it's generally more common to just prefix the test method name with test. Phpunit automatically picks that up as a test to run and not a helper method. So `testGetAllAdminAccounts()` would work here and helps with readability of the methods. I think that's what we use across the rest of the codebase so good to be consistent, but I'll defer to @acsauk on that.

Reply:
We a mixture of both looking at the codebase which obviously can be quite confusing to be honest. But yeah for this file we use the `test` prefix on the function rather than the `@test` annotation comment so it would be wise to update the new tests in here to be consistent with what's already in here 👍🏻

Reply:
Done!